guides/micronaut-error-handling/groovy/src/main/groovy/example/micronaut/BookController.groovy

```diff
@@ -1,6 +1,9 @@
+package example.micronaut
+/*
 //tag::package[]
 package example.micronaut
 //end::package[]
+*/
```

Comment:
I tried, but couldn't find anything better than this 😿

Reply:
Yes, that was the best I could come up with myself.